### DISCUSSION TITLE
Placeholder for Formatted balance without currency symbol

### DISCUSSION THREAD

**Discussion by CloudeLecaw** [Zrips/CMI#6556] [Enhancement]
We added a lot of ICs where we show the current balance in our format
our problem is that the currency sybol there should get an other color

so is it possible to add a placeholder for formatted balance without currently symbol?

**Comment by Zrips** [comment]
Placeholder returns formatted balance as defined in config file which can include currency symbol

**Comment by CloudeLecaw** [comment]
So how to exclude the currecy symbol if needed in some cases?

**Comment by CloudeLecaw** [comment]
closed because its no longer relevant :)
